feature-libs/user/account/components/myaccount-view-name/myaccount-view-name.module.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MyaccountViewNameComponent } from './myaccount-view-name.component';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  UrlModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        MyAccountViewNameComponent: {
+          component: MyaccountViewNameComponent,
+          guards: [AuthGuard],
+        },
+      },
+    }),
+  ],
+  declarations: [MyaccountViewNameComponent],
+  exports: [MyaccountViewNameComponent],
+  imports: [CommonModule, RouterModule, UrlModule, I18nModule],
+})
+export class MyaccountViewNameModule {}
```

Comment:
[MEDIUM]
My account are 2 words, so lets distinguish them by using camelCase/PascalCase for it when needed, e.g. `myAccount` / `MyAccount`. 
Please review all the places in this PR where `Myaccount` or `myaccount` phrases are used and please fix them